dashboard/app/models/school.rb

```diff
@@ -327,6 +353,31 @@ def self.seed_from_s3
           }
         end
       end
+
+      # Some of this data has #- appended to the front, so we strip that off with .to_s.slice(2) (it's always a single digit)
+      CDO.log.info "Seeding 2019-2020 public school data."
+      AWS::S3.seed_from_file('cdo-nces', "2019-2020/ccd/schools.csv") do |filename|
+        merge_from_csv(filename, {headers: true, quote_char: "\x00"}, true, is_dry_run: false) do |row|
+          row = row.to_h.map {|k, v| [k, sanitize_string_for_db(v)]}.to_h
+          {
+            id:                           row['School ID - NCES Assigned [Public School] Latest available year'].to_i.to_s,
+            name:                         row['School Name'].upcase,
+            address_line1:                row['Location Address 1 [Public School] 2019-20'].to_s.upcase.truncate(50).presence,
+            address_line2:                row['Location Address 2 [Public School] 2019-20'].to_s.upcase.truncate(30).presence,
+            address_line3:                row['Location Address 3 [Public School] 2019-20'].to_s.upcase.presence,
+            city:                         row['Location City [Public School] 2019-20'].to_s.upcase.presence,
+            state:                        row['Location State Abbr [Public School] 2019-20'].to_s.strip.upcase.presence,
+            zip:                          row['Location ZIP [Public School] 2019-20'],
+            latitude:                     row['Latitude [Public School] 2019-20'].to_f,
+            longitude:                    row['Longitude [Public School] 2019-20'].to_f,
+            school_type:                  CHARTER_SCHOOL_MAP[row['Charter School [Public School] 2019-20'].to_s] || 'public',
+            school_district_id:           row['Agency ID - NCES Assigned [Public School] Latest available year'].to_i,
+            state_school_id:              row['State School ID [Public School] 2019-20'],
+            school_category:              SCHOOL_CATEGORY_MAP[row['School Type [Public School] 2019-20']].presence,
+            last_known_school_year_open:  OPEN_SCHOOL_STATUSES.include?(row['Updated Status [Public School] 2019-20'].to_s.slice(2..-1)) ? '2019-2020' : nil
```

Comment:
Since we do an explicit map for other places like this instead of cutting off the first two characters, I'd prefer if we followed one pattern or the other?